_posts/2019-03-06-release-1.8.0.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title:  "Apache Flink 1.8.0 Release Announcement"
+date:   2019-2-26 12:00:00
+categories: news
+authors:
+- till:
+  name: "Aljoscha Krettek"
+  twitter: "aljoscha"
+---
+
+The Apache Flink community is pleased to announce Apache Flink 1.8.0.  The
+latest release includes more than 370 resolved issues and some exciting
+additions to Flink that we describe in the following sections of this post.
+Please check the [complete changelog](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12344274)
+for more details.
+
+
+Flink 1.8.0 is API-compatible with previous 1.x.y releases for APIs annotated
+with the `@Public` annotation.  The release is available now and we encourage
+everyone to [download the release](http://flink.apache.org/downloads.html) and
+check out the updated
+[documentation](https://ci.apache.org/projects/flink/flink-docs-release-1.8/).
+Feedback through the Flink [mailing
+lists](http://flink.apache.org/community.html#mailing-lists) or
+[JIRA](https://issues.apache.org/jira/projects/FLINK/summary) is, as always,
+very much appreciated!
+
+You can find the binaries on the updated [Downloads page](http://flink.apache.org/downloads.html) on the Flink project site.
+
+{% toc %}
+
+With Flink 1.8.0 we come closer to our goals of enabling fast data processing
+and building data-intensive applications for the Flink community in a seamless
+way. We do this by cleaning up and refactoring Flink under the hood to allow
+more efficient feature development in the future. This includes removal of the
+legacy runtime components that were subsumed in the major rework of Flinks
+underlying distributed system
+([FLIP-6](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=65147077))
+as well as refactorings on the Table API that prepare it for the future
+addition of the Blink enhancements
+([FLINK-11439](https://issues.apache.org/jira/browse/FLINK-11439)).
+
+Nevertheless, this release includes some important new features and bug fixes.
+The most interesting of those we'll highlight below. Please consult the
+[complete changelog](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12344274)
+or the [release notes](https://ci.apache.org/projects/flink/flink-docs-release-1.8/release-notes/flink-1.8.html)
+for more details.
+
+
+## New Features and Improvements
```

Comment:
Please add two new features:

**SQL pattern detection with user-defined functions and aggregations**:
The support of the `MATCH_RECOGNIZE` clause has been extended by multiple features. User-defined function are now possible which allows for custom logic during pattern detection [FLINK-10597]. Aggregations allow for more complex CEP definitions [FLINK-7599].

```
SELECT *
FROM Ticker
    MATCH_RECOGNIZE (
        ORDER BY rowtime
        MEASURES
            AVG(A.price) AS avgPrice
        ONE ROW PER MATCH
        AFTER MATCH SKIP TO FIRST B
        PATTERN (A+ B)
        DEFINE
            A AS AVG(A.price) < 15
    ) MR;
```

**RFC-compliant CSV format**:
The SQL tables can now be read and written in an RFC-4180 standard compliant CSV table format. The format might also be useful for general DataStream API users [FLINK-9964].

Reply:
Done